tests/foreman/ui_airgun/test_oscapcontent.py

```diff
@@ -0,0 +1,87 @@
+"""Tests for Oscapcontent
+
+:Requirement: Oscapcontent
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.config import settings
+from robottelo.datafactory import gen_string
+from robottelo.decorators import fixture, tier2
+
+
+@fixture(scope='module')
+def oscap_content_path():
+    return settings.oscap.content_path
+
+
+def test_positive_create(session, oscap_content_path):
```

Comment:
after last change, we can treat that test case as tier2, so if you work with that file in future, just add the tag and docstring, not a blocker atm